chain/arweave/src/adapter.rs

```diff
@@ -56,17 +62,35 @@ impl ArweaveTransactionFilter {
                     && !data_source.mapping.transaction_handlers.is_empty()
             })
             .map(|ds| {
-                base64_url::decode(&ds.source.owner.clone().unwrap_or_default()).unwrap_or_default()
+                let owner = ds.source.owner.clone().unwrap_or_default();
+
+                if owner.len() == HEX_SHA256_LEN {
+                    owner.into_bytes()
+                } else {
+                    base64_url::decode(&owner).unwrap_or_default()
```

Comment:
Are we not checking because the input has already be validated?

Reply:
actually this was already there but it's prolly a good idea to validate it